docs/tutorials/exploring_bdk_rn.md

```diff
@@ -0,0 +1,655 @@
+We want to see the response of this call so let's use `setDisplayText()` to see the output
+
+```javascript
+
+  const createWallet = async () => {
+     const { data } = await BdkRn.createWallet({
+      mnemonic: mnemonic,
+      password: 'password',
+      network: 'testnet',
+    });
+    setWallet(data);
+    setDisplayText(JSON.stringify(data));
+  };
+```
+
+A new button will be required to trigger `createWallet` 
+
+Let's add a new button just above `{/* input boxes and send transaction button */}` 
+
+```jsx
+				<Button
+            title="Create Wallet"
+            style={styles.methodButton}
+            onPress={createWallet}
+          />
+        </View>
+        {/* input boxes and send transaction button */}
+```
+
+The response returned by `createWallet` is a new address for the created wallet.
+
+```javascript
+{
+  "data": {
+    "address": "tb1qxg8g8cdzgs09cttu3y7lc33udqc4wsesunjnhe"
+  },
+  "error": false
+}
+```
+
+The AwesomeBitcoinApp should now be creating a wallet when we click **Create Mnemonic** followed by **Create Wallet**.
+
+<img src="./exploring_bdk_rn/bdk_rn_tutorial_screen_createwallet.png" style="display: block; margin: 0 auto; zoom:50%;" />
+
+The wallet created is a HD wallet and the address displayed is the 0 index address for the wallet.  The path used by default is 84'/1'/0'/0/* for addresses and 84'/1'/0'/1/* for change.
+
+As we specified `testnet` and did not specify `blockChainName` and  `blockChainConfigUrl` a default testnet  server will be used as the bitcoin node, `ssl://electrum.blockstream.info` is the default url used for testnet.
+
+Using `mnemonic` is a quick way to create a new wallet with bdk-rn. The `createWallet()` method in `bdk-rn` has many optional arguments to configure the wallet. In addition to mnemonic, a wallet can also be created with a descriptor. If a descriptor is passed as an argument the wallet will be created using the descriptor. When using a descriptor, arguments for network, password and mnemonic are not required. bdk-rn has a `createDescriptor()` method to create a descriptor. More about output descriptors here https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md and here https://github.com/LtbLightning/bdk-rn#createdescriptor
+
+```javascript
+// using a descriptor to create wallet 
+const response = await BdkRn.createWallet({
+  descriptor:
+  'tprv8ZgxMBicQKsPd3G66kPkZEuJZgUK9QXJRYCwnCtYLJjEZmw8xFjCxGoyx533AL83XFcSQeuVmVeJbZai5RTBxDp71Abd2FPSyQumRL79BKw'
+});
+```
+
+Other arguments for `createWallet()` are:
+
+**blockChainName**: Blockchain backend to use, like [`electrum` (opens new window)](https://github.com/romanz/electrs), [`esplora` (opens new window)](https://github.com/Blockstream/esplora), `compact-filters` ([BIP157 (opens new window)](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki)) and Bitcoin Core. bdk-rn at the moment doesn't support compact-filters and Bitcoin Core, this will be added shortly in a future release.
```

Comment:
```suggestion
**blockChainName**: Blockchain backend to use, like [`electrum`](https://github.com/romanz/electrs), [`esplora`](https://github.com/Blockstream/esplora), `compact-filters` ([BIP157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki)) and Bitcoin Core. bdk-rn at the moment doesn't support compact-filters and Bitcoin Core, this will be added shortly in a future release.
```

Reply:
Done